mesonbuild/scripts/depfixer.py

```diff
@@ -391,12 +342,9 @@ def remove_rpath_entry(self, entrynum: int) -> None:
         return None
 
 def fix_elf(fname: str, rpath_dirs_to_remove: T.Set[bytes], new_rpath: T.Optional[bytes], verbose: bool = True) -> None:
-    with Elf(fname, verbose) as e:
-        if new_rpath is None:
-            e.print_rpath()
-            e.print_runpath()
```

Comment:
I guess those print have been added for debugging, that could still be useful to have if we find bugs in that script. Maybe just comment out those instead?